src/actions/arcActions.ts

```diff
@@ -324,7 +325,7 @@ async function getProposalDetails(dao: Arc.DAO, votingMachineInstance: Arc.Genes
     votesYes: Util.fromWei(yesVotes).toNumber(),
     votesNo: Util.fromWei(noVotes).toNumber(),
     winningVote: Number(proposalDetails[9]),
-    threshold: Util.fromWei(await votingMachineInstance.getThreshold({avatar: dao.avatar.address, proposalId})).toNumber()
+    threshold: Util.fromWei(new BigNumber(await votingMachineInstance.getThreshold({avatar: dao.avatar.address, proposalId}))).toNumber()
```

Comment:
`getThreshold` should already be returning a BigNumber

Reply:
fixed: https://github.com/daostack/alchemy/pull/81/commits/ee837c9797978fc6976021364697bca1ff0c86d0